### PR TITLE
Update style-loader and mini-css-extract-plugin

### PR DESCRIPTION
**Pull request by jtpio**
## References

Addresses part of #9297

Follow-up to https://github.com/jupyterlab/jupyterlab/pull/9364

## Code changes

Dependency update.

## User-facing changes

None

## Backwards-incompatible changes

None

### CODE REVIEW

**Comment by jtpio** [comment]
Looks like a breaking change to fix:

image

**Comment by jtpio** [comment]
Documented in the changelog:

https://github.com/webpack-contrib/mini-css-extract-plugin/blob/master/CHANGELOG.md#-potential-breaking-change

> Just remove them from your configuration(s). We did this because many developers started making these mistakes.

File: packages/nbconvert-css/webpack.config.js [hunk 1]
```diff
@@ -18,10 +18,7 @@ module.exports = {
         test: /\.css$/,
         use: [
           {
-            loader: MiniCssExtractPlugin.loader,
-            options: {
-              hmr: process.env.NODE_ENV === 'development'
-            }
+            loader: MiniCssExtractPlugin.loader
```
Comment (jasongrout) [inline packages/nbconvert-css/webpack.config.js:21]: Now this can be:

```typescript
        use: [MiniCssExtractPlugin.loader, 'css-loader'],
```

Comment (jtpio) [reply packages/nbconvert-css/webpack.config.js:21]: Yes, thanks!

**Comment by jasongrout** [comment]
It's too bad the nbconvert-css index.css suffers from the same problems we were seeing in css-loader elsewhere - that the css is duplicated many times. For example, it seems that Lumino widget css is duplicated 228 times and Blueprint css is duplicated 101 times. I don't think that is in scope to fix here, but maybe something to think about for the future.

**Comment by jasongrout** [comment]
> It's too bad the nbconvert-css index.css suffers from the same problems we were seeing in css-loader elsewhere - that the css is duplicated many times. For example, it seems that Lumino widget css is duplicated 228 times and Blueprint css is duplicated 101 times. I don't think that is in scope to fix here, but maybe something to think about for the future.

Actually, it looks like the change might be easy. Can I push to your branch?

**Comment by jtpio** [comment]
>  Can I push to your branch?

Sure thing!

**Comment by jasongrout** [comment]
The old index.css file is 1,146,862 lines. The new one is 13,423 lines, so about 1% the size.

**Comment by jasongrout** [comment]
Note that our current published nbconvert-css package index.css is  13046 lines, 503K, so as expected, the deduplication was happening before. So this new commit is needed to make sure there is not a regression, and to bring things down to about the same size.

**Review by jasongrout — APPROVED** [review]
I approve @jtpio's changes.

**Comment by jasongrout** [comment]
@jtpio - if my changes look good to you, feel free to merge this.

**Comment by jtpio** [comment]
Thanks Jason for the extra fixes!

**Pull request merged**